src/gluonnlp/model/transformer.py

```diff
@@ -84,6 +87,10 @@ class BasePositionwiseFFN(HybridBlock):
     params : Parameter or None
         Container for weight sharing between cells.
         Created if `None`.
+    activation : str, default 'relu'
```

Comment:
activation was already mentioned in line 79

Reply:
ok, deleted